### ISSUE TITLE
Vscode emits a sound when suggst something with github copilot

### ISSUE THREAD

**Issue by DramaBooks** [microsoft/vscode#201994]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
- OS Version: 

Steps to Reproduce:

1. In an editor window, I digit something
2. Github copilot emits a sound and then suggests something.
3. I want eliminate this tedious sound. I'm using a screen reader, I cannot disable accessibility feature, I want only delete or change the sound.
4.

**Comment by meganrogge** [comment]
You can manage this by searching the settings editor for `audio cues` or by running the command `Help: List Audio cues` and tabbing to the settings gear.

**Comment by tinahmgao** [comment]
btw, seem there no such setting as audio cues, here's the search result, and turn this off doesn't turn off the sound. For now, running the Help command does the work.
![image](https://github.com/microsoft/vscode/assets/47345415/f1f48bb0-98e5-4241-8099-e1ecf3a87b23)

**Comment by meganrogge** [comment]
it is now called accessibility signals

**Comment by AsadullahAbbasi** [comment]
type >audio cues
![Image](https://github.com/user-attachments/assets/7dc3ec30-e66d-45b2-a2ad-d235d0b6c6a8)
then uncheck diff boxes including progress as you want by the way that tedious sound was of progree
